<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Christopher J.F. Cameron | Curriculum Vitae</title>
  <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

  <link rel="apple-touch-icon" sizes="152x152" href="https://ccameron.github.io/assets/img/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="https://ccameron.github.io/assets/img/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="https://ccameron.github.io/assets/img/favicon-16x16.png">
  <link rel="manifest" href="https://ccameron.github.io/assets/img/site.webmanifest">
  <link rel="mask-icon" href="https://ccameron.github.io/assets/img/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">
  <!-- <link rel="shortcut icon" type="image/png" href="https://ccameron.github.io/assets/img/favicon.png"> -->

  <link rel="stylesheet" href="https://ccameron.github.io/assets/css/main.css">
  <link rel="canonical" href="https://ccameron.github.io/CV/">
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        Christopher J.F. <strong>CAMERON</strong>
    </span>
    

    <nav class="site-nav">

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="https://ccameron.github.io/">About</a>

        <!-- Blog 
        <a class="page-link" href="https://ccameron.github.io/blog/">blog</a>-->

        <!-- Pages -->
        
          
            <a class="page-link" href="https://ccameron.github.io/CV/">CV</a>
          
        
          
        
          
        
          
            <a class="page-link" href="https://ccameron.github.io/teaching/">Teaching</a>
          
        
          
        

        <!-- CV link -->
        <!-- <a class="page-link" href="https://ccameron.github.io/assets/pdf/CV.pdf">vitae</a> -->

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">
      Curriculum Vitae
      
    </h1>
    <h5 class="post-description">Markdown/HTML and PDF formats of the CV may be found below.</h5>
  </header>

  <article class="post-content Curriculum Vitae clearfix">
    <p>For a PDF version of the CV, click here: <a href="/downloads/ChristopherCameron_CV.pdf">PDF</a></p>

<h3><strong>Christopher John Frederick Cameron</strong></h3>

<p><sup>1</sup> McGill School of Computer Science and Centre for Bioinformatics, McGill University<br />
<sup>2</sup> Department of Biochemistry and Goodman Cancer Research Centre, McGill University<br /><br />
With the advent of the Chromosome Conformation Capture (3C) technologies, the ability to quantify long-range interactions of genomic loci at high-resolution has become possible. My Ph.D. dissertation focuses on this biochemical technology (in particular, 3C Carbon-Copy [5C] and high-throughput [Hi-C]) by using novel implementations of statistical modeling, machine learning, and bioinformatics approaches to understand how cells regulate themselves through the organization of chromatin’s higher-order structures.<br /><br />
Keywords: bioinformatics, 4D regulatory genomics, AI specialist</p>
<h4><strong>Contact information</strong></h4>
<!-- christopher dot cameron at mail dot mcgill dot ca -->
<p><a href="mailto:christopher.cameron@mail.mcgill.ca">christopher [dot] cameron [at] mail [dot] mcgill [dot] ca</a>
<br />
<a href="https://ccameron.github.io/">https://ccameron.github.io/</a></p>

<h4><strong>Positions held</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="20%" />
		<col width="80%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019-Present</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Researcher</strong><br /><i>Department of Molecular Biophysics and Biochemistry, Yale University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Research Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Course Lecturer</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Teaching Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Graduate Service Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Java Developer</strong><br /><i>School of Environmental Sciences, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2008-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Research Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
</table>

<h4><strong>Education</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="20%" />
		<col width="80%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019-Present</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Researcher</strong><br />Yale University, New Haven, USA<br /><i>Advisors</i>: <a href="http://www.gersteinlab.org/">Mark Gerstein</a> and <a href="https://xiong.yale.edu/">Yong Xiong</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Ph.D. in Computer Science (conc. in Bioinformatics)</strong><br />McGill University, Montréal, Canada<br /><i>Thesis</i>: High-resolution computational analysis of chromatin architecture and function<br /><i>Advisors</i>: <a href="http://www.cs.mcgill.ca/~blanchem/">Mathieu Blanchette</a> and <a href="https://www.researchgate.net/profile/Josee_Dostie">Josée Dostie</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2014</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>M.Sc. in Bioinformatics</strong> (February 2014)<br />University of Guelph, Guelph, Canada<br /><i>Thesis</i>: <a href="https://atrium.lib.uoguelph.ca/xmlui/handle/10214/7854">Tissue-to-plasma Partition Coefficient Prediction by a Multi-channel Restricted Boltzmann Machine</a><br /><i>Advisors</i>: <a href="https://uwaterloo.ca/pharmacokinetics-research-group/">Andrea Edginton</a>, <a href="https://ovc.uoguelph.ca/biomedical-sciences/people/faculty/Ronald-Johnson">Ronald Johnson</a>, and <a href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2006-2012</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>B.Sc. in Biomedical Toxicology</strong> (February 2012)<br />University of Guelph, Guelph, Canada<br /><i>Thesis</i>: Molecular Graph Neural Networks for Toxicology<br /><i>Advisor</i>: <a href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
</table>

<h4><strong>Journal publications</strong></h4>
<p>*co-first authors</p>
<ol reversed="">
	<li>Bonetti A., Agostini F., Hashimoto K., Suzuki A.M., Pascarella G., Gimenez J., Arner E., <u>Cameron C.J.F.</u>, et al. (2018) <strong>RADICL-seq, a novel methodology for identifying genome-wide RNA-chromatin interactions</strong>. <i>In preparation for submission to Nature Genetics</i>.</li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2018) <strong>Estimating DNA-DNA interaction frequency from Hi-C data at restriction-fragment resolution</strong>. <i>In review at Genome Biology</i>. doi:<a href="https://doi.org/10.1101/377523"> 10.1101/377523</a></li>
	<li>Wang X.Q.D.*, <u>Cameron C.J.F.</u>*, Paquette D., Segal D., Warsaba R., Blanchette M., and Dostie J. (2019) <strong>2C-ChIP: measuring chromatin immunoprecipitation signal from defined genomic regions with deep sequencing</strong>. <i>BMC Genomics</i>, <strong>20</strong>:162. doi:<a href="https://doi.org/10.1186/s12864-019-5532-5">10.1186/s12864-019-5532-5</a></li>
	<li>Roche P.J.R., Gytz H., Hussain F., <u>Cameron C.J.F.</u>, Paquette D., Blanchette M., Dostie J., Nagar B., and Akavia U.D. (2018) <strong>Double-Stranded Biotinylated Donor Enhances Homology-Directed Repair in Combination with Cas9 Monoavidin in Mammalian Cells</strong>. <i>The CRISPR Journal</i>, <strong>1</strong>(6). doi:<a href="https://doi.org/10.1089/crispr.2018.0045">10.1089/crispr.2018.0045</a> </li>
	<li>Malina A. <u>Cameron C.J.F.</u>, Robert F., Blanchette M., Dostie J., and Pelletier J. (2015) <strong>PAM multiplicity marks genomic target sites as inhibitory to CRISPR/Cas9 editing</strong>. <i>Nature Communications</i>, <strong>6</strong>:10124. doi:<a href="https://doi.org/10.1038/ncomms10124">10.1038/ncomms10124</a></li>
	<li>Ma E.Y.T., <u>Cameron C.J.F.</u>, and Kremer S.C. (2010) <strong>Classifying and scoring of molecules with the NGN: new data sets, significance tests, and generalization</strong>. <i>BMC Bioinformatics</i>, <strong>11</strong>(Suppl 8):S4. doi:<a href="https://doi.org/10.1186/1471-2105-11-S8-S4">10.1186/1471-2105-11-S8-S4</a></li>
</ol>

<h4><strong>Peer-reviewed conference papers</strong></h4>
<ol reversed="">
	<li><u>Cameron C.J.F.</u>, Ma E.Y.T., and Kremer S.C. (2010) <strong>Neural Grammar Networks for Toxicology</strong>. <i>In Computational Intelligence in Bioinformatics and Computational Biology (CIBCB), 2010 IEEE Symposium on</i>: 1-8. doi:<a href="https://doi.org/10.1109/CIBCB.2010.5510322">10.1109/CIBCB.2010.5510322</a></li>
</ol>

<h4><strong>Conferences with abstract submission</strong></h4>
<ol reversed="">
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2018) <strong>Prediction of complete Hi-C interaction matrices from sequence-based determinants</strong>. <i>IRIC 2018 Annual Symposium</i>, Montréal, Canada </li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence</strong>. <i>SCS and RegSys COSI at ISMB 2018</i>, Chicago, USA</li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence alone</strong>. <i>RECOMB 2018</i>, Paris, France</li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2017) <strong>High-resolution estimation of true DNA-DNA interaction frequency from Hi-C data</strong>. <i>SCS and RegGen COSI at ISMB/ECCB 2017</i>, Prague, Czech Republic</li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>MRCCT - 1st International Symposium on Immunogenetics of Infectious and Inflammatory Diseases</i>, Montréal, Canada (Ph.D. poster prize)</li>
	<li><u>Cameron C.J.F.</u>, Blanchette M., De Hoon M., and Dostie J. (2016) <strong>Identifying ncRNA Drivers of Architectural Change in Chromatin</strong>. <i>RNA2016</i>, Kyoto, Japan</li>
	<li><u>Cameron C.J.F.</u>, Dostie J., and Blanchette M. (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>GLBIO/CCBC 2016</i>, Toronto, Canada</li>
	<li><u>Cameron C.J.F.</u>, Kaplan M., Drouin A., and Blanchette M. (2016) <strong>Linking Transposable Elements to Chromatin Architecture in <i>Arabidopsis thaliana</i></strong>. <i>GLBIO/CCBC 2016</i>, Toronto, Canada</li>
	<li><u>Cameron C.J.F.</u>, Fraser J., Dostie J., and Blanchette M. (2014) <strong>Chromosome conformation capture data improves long-range eQTL prediction</strong>. <i>2014 CSHL Conference on Epigenetics &amp; Chromatin</i>, Huntington, USA</li>
</ol>
<!-- <br>&nbsp;<br> -->

<h4><strong>Book chapters, reviews, and non-refereed publications</strong></h4>
<ol reversed="">
	<li><u>Cameron C.J.F.</u>, Fraser J., Blanchette M., and Dostie J. (2016) <strong>Mapping and Visualizing Spatial Genome Organization. In: Bazett-Jones D., Dellaire G. (eds) The Functional Nucleus</strong>. <i>Springer, Cham</i>. doi:<a href="https://doi.org/10.1007/978-3-319-38882-3_16">10.1007/978-3-319-38882-3_16</a></li>
	<li>Marhon S.A., <u>Cameron C.J.F.</u>, and Kremer S.C. (2013) <strong>Recurrent Neural Networks. In: Bianchini M., Maggini M., Jain L. (eds) Handbook on Neural Information Processing</strong>. <i>Intelligent Systems Reference Library, vol 49. Springer, Berlin, Heidelberg</i>. doi:<a href="https://doi.org/10.1007/978-3-642-36657-4_2">10.1007/978-3-642-36657-4_2</a></li>
</ol>

<h4><strong>Invited talks</strong></h4>
<ol reversed="">
	<li>(July 2017) <strong>The future of Hi-C</strong>. <i>Birds of a Feather panel discussion at ISMB/ECCB 2017</i>, Prague, Czech Republic</li>
	<li>(April 2017) <strong>High-resolution estimation of DNA-DNA contact frequency from Hi-C data</strong>. <i>The Physical Basis of Functional Genome Organization: Genome organization as viewed by molecular and visual techniques</i>, Holetown, Barbados</li>
	<li>(August 2016) <strong>The 3D Genome</strong>. <i>Beijing Institute of Genomics (BIG) at the Chinese Academy of Sciences</i>, Beijing, China</li>
</ol>

<h4><strong>Awards and scholarships</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="20%" />
		<col width="80%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Sumner Fellowship, <i>Walter Sumner Foundation</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2018</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Travel Fellowship Award for ISCB Student Council Symposium, <i>ISCB SCS</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Research Enhancement and Travel Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Mobility Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2016</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Mitacs Globalink Research Award - Japan Society for the Promotion of Science (JSPS), <i>Mitacs/JSPS</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2014</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Grad Excellence Award in Computer Science, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2014</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Hitschfeld Award - Computer Science, <i>McGill University</i></td>
	</tr>
</table>

<h4><strong>Summary of teaching activities</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="20%" />
		<col width="80%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>Fall 2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">COMP 364: Computer Tools for Life Sciences, <i>McGill University</i> (with <a href="http://cgoliver.com/">C.G. Oliver</a>)</td>
	</tr>
</table>

  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper" style="width: 100%;text-align: center;" >
    &copy; Copyright 2019 Christopher J.F. Cameron.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme.

    
  </div>

</footer>


    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="https://ccameron.github.io/assets/js/common.js"></script>





<!-- Include custom icon fonts -->
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/font-awesome.min.css">
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-111964338-1', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
