<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title>Christopher JF Cameron | Curriculum Vitae</title>
  <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

  <link rel="apple-touch-icon" sizes="152x152" href="https://ccameron.github.io/assets/img/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="https://ccameron.github.io/assets/img/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="https://ccameron.github.io/assets/img/favicon-16x16.png">
  <link rel="manifest" href="https://ccameron.github.io/assets/img/site.webmanifest">
  <link rel="mask-icon" href="https://ccameron.github.io/assets/img/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">
  <!-- <link rel="shortcut icon" type="image/png" href="https://ccameron.github.io/assets/img/favicon.png"> -->

  <link rel="stylesheet" href="https://ccameron.github.io/assets/css/main.css">
  <link rel="canonical" href="https://ccameron.github.io/CV/">
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        Christopher JF <strong>CAMERON</strong>
    </span>
    

    <nav class="site-nav">

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="https://ccameron.github.io/">About</a>

        <!-- Blog 
        <a class="page-link" href="https://ccameron.github.io/blog/">blog</a>-->

        <!-- Pages -->
        
          
            <a class="page-link" href="https://ccameron.github.io/CV/">CV</a>
          
        
          
        
          
        
          
        
          
        
          
            <a class="page-link" href="https://ccameron.github.io/teaching/">Teaching</a>
          
        
          
        

        <!-- CV link -->
        <!-- <a class="page-link" href="https://ccameron.github.io/assets/pdf/CV.pdf">vitae</a> -->

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">
      Curriculum Vitae
      
    </h1>
    <h5 class="post-description">Markdown/HTML and PDF formats of the CV may be found below.</h5>
  </header>

  <article class="post-content Curriculum Vitae clearfix">
    <p>For a PDF version of the CV, click here: <a target="_blank" href="/downloads/ChristopherJohnFrederickCameron_CV.pdf">PDF</a></p>

<h3><strong>Christopher John Frederick Cameron, PhD</strong></h3>

<p><sup>1</sup> Department of Molecular Biophysics and Biochemistry, Yale University</p>

<p style="text-align: justify"> As a <a target="_blank" href="https://en.wikipedia.org/wiki/Computational_biology">computational biologist</a>, I develop novel statistical, machine learning, and bioinformatics approaches to emerging <a target="_blank" href="https://en.wikipedia.org/wiki/Molecular_biology">molecular biology</a> technologies. My postdoctoral research focuses on <a target="_blank" href="https://en.wikipedia.org/wiki/Cryogenic_electron_microscopy">cryogenic Electron Microscopy (cryo-EM)</a>, specifically addressing the modeling and integration of prior information. During my PhD, I implemented analyses of  <a target="_blank" href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture">Chromosome Conformation Capture (3C)</a> and <a target="_blank" href="https://en.wikipedia.org/wiki/Chromatin_immunoprecipitation">Chromatin ImmunoPrecipitation (ChIP)</a> data that provided insight into the regulation of genomes through the organization of chromatin’s higher-order structures. 
</p>

<!-- <p style="text-align: justify">I am a <a target="_blank" href="https://en.wikipedia.org/wiki/Computational_biology">computational biologist</a> interested in developing novel analytical methods for emerging <a target="_blank" href="https://en.wikipedia.org/wiki/Molecular_biology">molecular biology</a> technologies. My PhD focused on applying innovative implementations of statistical modeling, machine learning, and bioinformatics approaches to <a target="_blank" href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture">Chromosome Conformation Capture (3C)</a> data (in particular, 3C <a target="_blank" href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture#5C_(many-vs-many)">Carbon-Copy [5C]</a> and <a target="_blank" href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture#Hi-C_(all-vs-all)">high-throughput [Hi-C]</a>). The resulting analyses provide insight into the regulation of genomes through the organization of chromatin’s higher-order structures (e.g., <a target="_blank" href="https://en.wikipedia.org/wiki/Topologically_associating_domain">Topologically Associating Domains [TAD]</a>).</p> -->
<p>Keywords: AI specialist, bioinformatics, 4D regulatory genomics, structural biology</p>

<h4><strong>Contact information</strong></h4>
<p><a href="mailto:christopher.cameron@yale.edu">christopher [dot] cameron [at] yale [dot] edu</a>
<br />
<a href="https://ccameron.github.io/">https://ccameron.github.io/</a></p>

<h4><strong>Education</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019-Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Researcher</strong><br />Yale University, New Haven, USA<br /><i>Advisors</i>: <a target="_blank" href="http://www.gersteinlab.org/">Mark Gerstein</a> and <a target="_blank" href="https://xiong.yale.edu/">Yong Xiong</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>PhD in Computer Science, concentration in Bioinformatics</strong> (October 2019)<br />McGill University, Montréal, Canada<br /><i>Thesis</i>: High-resolution computational analysis of chromatin architecture and function<br /><i>Advisors</i>: <a target="_blank" href="http://www.cs.mcgill.ca/~blanchem/">Mathieu Blanchette</a> and <a target="_blank" href="https://www.researchgate.net/profile/Josee_Dostie">Josée Dostie</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>MSc in Bioinformatics</strong> (February 2014)<br />University of Guelph, Guelph, Canada<br /><i>Thesis</i>: <a target="_blank" href="https://atrium.lib.uoguelph.ca/xmlui/handle/10214/7854">Tissue-to-plasma partition coefficient prediction by a multi-channel restricted boltzmann machine</a><br /><i>Advisors</i>: <a target="_blank" href="https://uwaterloo.ca/pharmacokinetics-research-group/">Andrea Edginton</a>, <a target="_blank" href="https://ovc.uoguelph.ca/biomedical-sciences/people/faculty/Ronald-Johnson">Ronald Johnson</a>, and <a target="_blank" href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2006-2011</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>BSc in Biomedical Toxicology, minor in Computer Science</strong> (February 2012)<br />University of Guelph, Guelph, Canada<br /><i>Thesis</i>: Molecular graph neural networks for toxicology<br /><i>Advisor</i>: <a target="_blank" href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
</table>

<h4><strong>Positions held</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019-Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Associate</strong><br /><i>Department of Molecular Biophysics and Biochemistry, Yale University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Graduate Research Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Course Lecturer</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Teaching Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Graduate Service Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Java Developer</strong><br /><i>School of Environmental Sciences, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2008-2013</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Research Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
</table>

<!-- <br>&nbsp;<br> -->
<h4><strong>Journal publications</strong></h4>
<p><sup>†</sup> co-first authors</p>
<ol reversed="">
	<li>Prost JA, <u>Cameron CJF</u>, and Blanchette M. <strong>Identifying sequence-based determinants of chromosomal compartments</strong>. <i>In preparation for submission to Genome Biol</i>.</li>
	<li><u>Cameron CJF</u>, Wang XQD, Dostie J, and Blanchette M. <strong>LAMPS: an analysis pipeline for sequence-specific ligation-mediated amplification reads</strong>. <i>In submission at GPB</i>.</li>
	<li>Leng J, <u>Cameron CJF</u>, Oh S, Khurana E, Noonan JP, and Gerstein MB. <strong>LESSeq: Local event-based analysis of alternative splicing using RNA-Seq data</strong>. <i>In submission at PLoS One</i>. doi:<a target="_blank" href="https://doi.org/10.1101/841494">10.1101/841494</a></li>
	<li>Ramilowski J, Yip CW, Agrawal S, Chang J, Ciani Y, Kulakovskiy IV, Mendez M, Ching Ooi JL, Ouyang JF, Parkinson N, Petri A, Roos L, Severin J, Yasuzawa K, Abugessaisa I, Akalin A, Antonov I, Arner E, Bonetti A, Bono H, Borsari B, Brombacher F, Cannistraci CV, <u>Cameron CJF</u> et al. <strong>Functional Annotation of Human Long Non-Coding RNAs via Molecular Phenotyping</strong>. <i>In submission at Genome Res</i>. doi:<a target="_blank" href="https://doi.org/10.1101/700864">10.1101/700864</a></li>
	<li>Bonetti A, Agostini F, Suzuki AM, Hashimoto K, Pascarella G, Gimenez J, Ross L, Nash A, Ghilotti M, Arner E, <u>Cameron CJF</u> et al. <strong>RADICL-seq identifies general and cell type-specific principles of genome-wide RNA-chromatin interactions</strong>. <i>In press at Nat Commun</i>. doi:<a target="_blank" href="https://doi.org/10.1101/681924">10.1101/681924</a></li>
	<li>Wang XQD, <u>Cameron CJF</u>, Segal D, Paquette D, Blanchette M, and Dostie J. <strong>Profiling chromatin landscape at high resolution and throughput with 2C-ChIP</strong>. <i>In press at Methods Mol Biol</i>.</li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2020) <strong>HIFI: estimating DNA-DNA interaction frequency from Hi-C data at restriction-fragment resolution</strong>. <i>Genome Biol</i>, <strong>21</strong>(1):11. doi:<a target="_blank" href="https://doi.org/10.1186/s13059-019-1913-y">10.1186/s13059-019-1913-y</a></li>
	<li>Wang XQD<sup>&dagger;</sup>, <u>Cameron CJF</u><sup>&dagger;</sup>, Paquette D, Segal D, Warsaba R, Blanchette M, and Dostie J (2019) <strong>2C-ChIP: measuring chromatin immunoprecipitation signal from defined genomic regions with deep sequencing</strong>. <i>BMC Genomics</i>, <strong>20</strong>:162. doi:<a target="_blank" href="https://doi.org/10.1186/s12864-019-5532-5">10.1186/s12864-019-5532-5</a></li>
	<li>Roche PJR, Gytz H, Hussain F, <u>Cameron CJF</u>, Paquette D, Blanchette M, Dostie J, Nagar B, and Akavia UD (2018) <strong>Double-Stranded Biotinylated Donor Enhances Homology-Directed Repair in Combination with Cas9 Monoavidin in Mammalian Cells</strong>. <i>CRISPR J</i>, <strong>1</strong>(6). doi:<a target="_blank" href="https://doi.org/10.1089/crispr.2018.0045">10.1089/crispr.2018.0045</a> </li>
	<li>Malina A, <u>Cameron CJF</u>, Robert F, Blanchette M, Dostie J, and Pelletier J (2015) <strong>PAM multiplicity marks genomic target sites as inhibitory to CRISPR/Cas9 editing</strong>. <i>Nat Commun</i>, <strong>6</strong>:10124. doi:<a target="_blank" href="https://doi.org/10.1038/ncomms10124">10.1038/ncomms10124</a></li>
	<li>Ma EYT, <u>Cameron CJF</u>, and Kremer SC (2010) <strong>Classifying and scoring of molecules with the NGN: new data sets, significance tests, and generalization</strong>. <i>BMC Bioinformatics</i>, <strong>11</strong>(Suppl 8):S4. doi:<a target="_blank" href="https://doi.org/10.1186/1471-2105-11-S8-S4">10.1186/1471-2105-11-S8-S4</a></li>
</ol>

<h4><strong>Peer-reviewed conference papers</strong></h4>
<ol reversed="">
	<li><u>Cameron CJF</u>, Ma EYT, and Kremer SC (2010) <strong>Neural Grammar Networks for Toxicology</strong>. <i>2010 IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology</i> (Montréal, Canada): 1-8. doi:<a target="_blank" href="https://doi.org/10.1109/CIBCB.2010.5510322">10.1109/CIBCB.2010.5510322</a></li>
</ol>

<h4><strong>Conferences with abstract submission</strong></h4>
<ol reversed="">
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2018) <strong>Prediction of complete Hi-C interaction matrices from sequence-based determinants</strong>. <i>IRIC 2018 Annual Symposium</i>, Montréal, Canada </li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence</strong>. <i>SCS and RegSys COSI at ISMB 2018</i>, Chicago, USA</li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence alone</strong>. <i>RECOMB 2018</i>, Paris, France</li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2017) <strong>High-resolution estimation of true DNA-DNA interaction frequency from Hi-C data</strong>. <i>SCS and RegGen COSI at ISMB/ECCB 2017</i>, Prague, Czech Republic</li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>MRCCT - 1st International Symposium on Immunogenetics of Infectious and Inflammatory Diseases</i>, Montréal, Canada (PhD poster prize)</li>
	<li><u>Cameron CJF</u>, Blanchette M, De Hoon M, and Dostie J (2016) <strong>Identifying ncRNA Drivers of Architectural Change in Chromatin</strong>. <i>RNA2016</i>, Kyoto, Japan</li>
	<li><u>Cameron CJF</u>, Dostie J, and Blanchette M (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>GLBIO/CCBC 2016</i>, Toronto, Canada</li>
	<li><u>Cameron CJF</u>, Kaplan M, Drouin A, and Blanchette M (2016) <strong>Linking Transposable Elements to Chromatin Architecture in <i>Arabidopsis thaliana</i></strong>. <i>GLBIO/CCBC 2016</i>, Toronto, Canada</li>
	<li><u>Cameron CJF</u>, Fraser J, Dostie J, and Blanchette M (2014) <strong>Chromosome conformation capture data improves long-range eQTL prediction</strong>. <i>2014 CSHL Conference on Epigenetics &amp; Chromatin</i>, Huntington, USA</li>
</ol>

<h4><strong>Book chapters, reviews, and non-refereed publications</strong></h4>
<ol reversed="">
	<li><u>Cameron CJF</u>, Fraser J, Blanchette M, and Dostie J (2016) <strong>Mapping and Visualizing Spatial Genome Organization. In: Bazett-Jones D, Dellaire G (eds) The Functional Nucleus</strong>. <i>Springer, Cham</i>. doi:<a target="_blank" href="https://doi.org/10.1007/978-3-319-38882-3_16">10.1007/978-3-319-38882-3_16</a></li>
	<li>Marhon SA, <u>Cameron CJF</u>, and Kremer SC (2013) <strong>Recurrent Neural Networks. In: Bianchini M, Maggini M, Jain L (eds) Handbook on Neural Information Processing</strong>. <i>Intelligent Systems Reference Library, vol 49. Springer, Berlin, Heidelberg</i>. doi:<a target="_blank" href="https://doi.org/10.1007/978-3-642-36657-4_2">10.1007/978-3-642-36657-4_2</a></li>
</ol>

<h4><strong>Invited talks</strong></h4>
<ol reversed="">
	<li>(July 2017) <strong>The future of Hi-C</strong>. <i>Birds of a Feather panel discussion at ISMB/ECCB 2017</i>, Prague, Czech Republic</li>
	<li>(April 2017) <strong>High-resolution estimation of DNA-DNA contact frequency from Hi-C data</strong>. <i>The Physical Basis of Functional Genome Organization: Genome organization as viewed by molecular and visual techniques</i>, Holetown, Barbados</li>
	<li>(August 2016) <strong>The 3D Genome</strong>. <i>Beijing Institute of Genomics (BIG) at the Chinese Academy of Sciences</i>, Beijing, China</li>
</ol>

<h4><strong>Awards and scholarships</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017-2019</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Sumner Fellowship, <i>Walter Sumner Foundation</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2018</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Travel Fellowship Award for ISCB Student Council Symposium, <i>ISCB SCS</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Research Enhancement and Travel Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Mobility Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2016</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Mitacs Globalink Research Award - Japan Society for the Promotion of Science (JSPS), <i>Mitacs/JSPS</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2014</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Grad Excellence Award in Computer Science, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013-2014</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Hitschfeld Award - Computer Science, <i>McGill University</i></td>
	</tr>
</table>

<h4><strong>Summary of teaching activities</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>Fall 2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">COMP 364: Computer Tools for Life Sciences, <i>McGill University</i> (with <a target="_blank" href="http://cgoliver.com/">Oliver CG</a>)</td>
	</tr>
</table>

  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper" style="width: 100%;text-align: center;" >
    &copy; Copyright 2020 Christopher JF Cameron.
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme.

    
  </div>

</footer>


    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="https://ccameron.github.io/assets/js/common.js"></script>





<!-- Include custom icon fonts -->
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/font-awesome.min.css">
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/academicons.min.css">

<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-111964338-1', 'auto');
ga('send', 'pageview');
</script>


  </body>

</html>
