<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge">

  <title> | Curriculum Vitae</title>
  <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

  <link rel="apple-touch-icon" sizes="152x152" href="https://ccameron.github.io/assets/img/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="https://ccameron.github.io/assets/img/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="https://ccameron.github.io/assets/img/favicon-16x16.png">
  <link rel="manifest" href="https://ccameron.github.io/assets/img/site.webmanifest">
  <link rel="mask-icon" href="https://ccameron.github.io/assets/img/safari-pinned-tab.svg" color="#5bbad5">
  <meta name="msapplication-TileColor" content="#da532c">
  <meta name="theme-color" content="#ffffff">
  <!-- <link rel="shortcut icon" type="image/png" href="https://ccameron.github.io/assets/img/favicon.png"> -->

  <link rel="stylesheet" href="https://ccameron.github.io/assets/css/main.css">
  <link rel="canonical" href="https://ccameron.github.io/CV/">
  
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-111964338-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-111964338-1');
  </script>
  
</head>


  <body>

    <header class="site-header">

  <div class="wrapper">

    
    <span class="site-title">
        
        Christopher JF <strong>CAMERON</strong>
    </span>
    

    <nav class="site-nav">

      <div class="trigger">
        <!-- About -->
        <a class="page-link" href="https://ccameron.github.io/">About</a>

        <!-- Blog 
        <a class="page-link" href="https://ccameron.github.io/blog/">blog</a>-->

        <!-- Pages -->
        
          
            <a class="page-link" href="https://ccameron.github.io/CV/">CV</a>
          
        
          
        
          
        
          
        
          
        
          
            <a class="page-link" href="https://ccameron.github.io/teaching/">Teaching</a>
          
        
          
        

        <!-- CV link -->
        <!-- <a class="page-link" href="https://ccameron.github.io/assets/pdf/CV.pdf">vitae</a> -->

      </div>
    </nav>

  </div>

</header>



    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">
      Curriculum Vitae
      
    </h1>
    <h5 class="post-description">Markdown/HTML and PDF formats of the CV are found below.</h5>
  </header>

  <article class="post-content Curriculum Vitae clearfix">
    <p>For a PDF version of the CV, please click here: <a target="_blank" href="/downloads/ChristopherJohnFrederickCameron_CV.pdf">PDF</a></p>

<h3><strong>Christopher John Frederick Cameron, PhD</strong></h3>

<p><sup>1</sup> Department of Molecular Biophysics and Biochemistry, Yale University<br />
<sup>2</sup> Department of Radiology and Biomedical Imaging, Yale University</p>

<p style="text-align: justify"> As a <a target="_blank" href="https://en.wikipedia.org/wiki/Computational_biology">computational biologist</a>, I develop novel statistical, machine learning, and bioinformatics approaches to emerging <a target="_blank" href="https://en.wikipedia.org/wiki/Molecular_biology">molecular biology</a> technologies. My postdoctoral research focuses on <a target="_blank" href="https://en.wikipedia.org/wiki/Cryogenic_electron_microscopy">cryogenic Electron Microscopy (cryo-EM)</a> image processing, specifically addressing the modeling and integration of prior information. During my PhD, I implemented analyses of  <a target="_blank" href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture">Chromosome Conformation Capture (3C)</a> and <a target="_blank" href="https://en.wikipedia.org/wiki/Chromatin_immunoprecipitation">Chromatin ImmunoPrecipitation (ChIP)</a> data that provided insight into the regulation of genomes through the organization of chromatin’s higher-order structures. 
</p>

<p>Keywords: bioinformatics, AI specialist, 4D regulatory genomics, structural biology</p>

<h4><strong>Contact information</strong></h4>
<p><a href="mailto:christopher.cameron&#64;yale.edu">christopher [dot] cameron [at] yale [dot] edu</a>
<br />
<a href="https://ccameron.github.io/">https://ccameron.github.io/</a></p>

<h4><strong>Education</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019&ndash;Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Researcher</strong><br />Yale University, New Haven, USA<br /><i>Advisors</i>: <a target="_blank" href="http://www.gersteinlab.org/">Mark Gerstein</a>, <a target="_blank" href="http://noodle.med.yale.edu/hdtag/profile.html">Hemant Tagare</a> and <a target="_blank" href="https://xiong.yale.edu/">Yong Xiong</a> </td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013&ndash;19</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>PhD in Computer Science, concentration in Bioinformatics</strong> (October 2019)<br />McGill University, Montréal, CAN<br /><i>Thesis</i>: <a target="_blank" href="https://escholarship.mcgill.ca/concern/theses/r207tt73g">High-resolution computational analysis of chromatin architecture and function</a><br /><i>Advisors</i>: <a target="_blank" href="http://www.cs.mcgill.ca/~blanchem/">Mathieu Blanchette</a> and <a target="_blank" href="https://www.researchgate.net/profile/Josee_Dostie">Josée Dostie</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012&ndash;13</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>MSc in Bioinformatics</strong> (February 2014)<br />University of Guelph, Guelph, CAN<br /><i>Thesis</i>: <a target="_blank" href="https://atrium.lib.uoguelph.ca/xmlui/handle/10214/7854">Tissue-to-plasma partition coefficient prediction by a multi-channel restricted Boltzmann machine</a><br /><i>Advisors</i>: <a target="_blank" href="https://uwaterloo.ca/pharmacokinetics-research-group/">Andrea Edginton</a>, <a target="_blank" href="https://ovc.uoguelph.ca/biomedical-sciences/people/faculty/Ronald-Johnson">Ronald Johnson</a> and <a target="_blank" href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2006&ndash;11</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>BSc in Biomedical Toxicology, minor in Computer Science</strong> (February 2012)<br />University of Guelph, Guelph, CAN<br /><i>Thesis</i>: Molecular graph neural networks for toxicology<br /><i>Advisor</i>: <a target="_blank" href="https://www.researchgate.net/profile/Stefan_Kremer">Stefan Kremer</a></td>
	</tr>
</table>

<h4><strong>Positions held</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2019&ndash;Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Postdoctoral Associate</strong><br /><i>Department of Molecular Biophysics and Biochemistry, Yale University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013&ndash;19</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Graduate Research Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Course Lecturer</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013&ndash;17</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Teaching Assistant</strong><br /><i>School of Computer Science, McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012&ndash;13</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Graduate Service Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2012&ndash;13</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Java Developer</strong><br /><i>School of Environmental Sciences, University of Guelph</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2008&ndash;13</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><strong>Research Assistant</strong><br /><i>School of Computer Science, University of Guelph</i></td>
	</tr>
</table>

<!-- &nbsp; -->
<h4><strong>Journal publications</strong></h4>
<p><sup>†</sup> co-first or -second authors</p>
<ol reversed="">
	<li>
		Leng J, <u>Cameron CJF</u>, Oh S, Khurana E, Noonan JP and Gerstein MB. 
		<strong>LESSeq: local event-based analysis of alternative splicing using RNA-Seq data</strong>. 
		<i>BioRxiv, in preparation</i>. 
		DOI:<a target="_blank" href="https://doi.org/10.1101/841494">10.1101/841494</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1101/841494" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Rozowsky J, Drenkow J, Yang YT, [and 96 others, including <u>Cameron CJF</u>]. <strong>Multi-tissue integrative analysis of personal epigenomes</strong>. <i>In submission at Science</i>. DOI:<a target="_blank" href="https://doi.org/10.1101/2021.04.26.441442">10.1101/2021.04.26.441442</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1101/2021.04.26.441442" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Amaya C, <u>Cameron CJF</u><sup>&dagger;</sup>, Devarkar SC<sup>&dagger;</sup>, Seager SJH, Gerstein MB, Xiong Y and Schlieker C. 
		<strong>Nodal modulator is required to sustain endoplasmic reticulum morphology</strong>. 
		<i>In submission at eLife</i>. 
		DOI:<a target="_blank" href="https://doi.org/10.1101/2021.01.28.428001">10.1101/2021.01.28.428001</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1101/2021.01.28.428001" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Prost JA<sup>&dagger;</sup>, <u>Cameron CJF</u><sup>&dagger;</sup> and Blanchette M. 
		<strong>SACSANN: identifying sequence-based determinants of chromosomal compartments</strong>. 
		<i>In submission at PLoS Comput Biol</i>. 
		DOI:<a target="_blank" href="https://doi.org/10.1101/2020.10.06.328039">10.1101/2020.10.06.328039</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1101/2020.10.06.328039" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Ramilowski J, Yip CW, Agrawal S, [and 115 others, including <u>Cameron CJF</u>] (2020) 
		<strong>Functional Annotation of Human Long Non-Coding RNAs via Molecular Phenotyping</strong>. 
		<i>Genome Res</i>. 
		DOI:<a target="_blank" href="https://doi.org/10.1101/gr.254219.119">10.1101/gr.254219.119</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1101/gr.254219.119" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		<u>Cameron CJF</u>, Wang XQD, Dostie J and Blanchette M (2020) 
		<strong>LAMPS: an analysis pipeline for sequence-specific ligation-mediated amplification reads</strong>. 
		<i>BMC Res Notes</i>, <strong>13</strong>:273. 
		DOI:<a target="_blank" href="https://doi.org/10.1186/s13104-020-05106-1">10.1186/s13104-020-05106-1</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1186/s13104-020-05106-1" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Bonetti A, Agostini F, Suzuki AM, [and 26 others, including <u>Cameron CJF</u>] (2020) 
		<strong>RADICL-seq identifies general and cell type-specific principles of genome-wide RNA-chromatin interactions</strong>. 
		<i>Nat Commun</i>, <strong>11</strong>:1018. 
		DOI:<a target="_blank" href="https://doi.org/10.1038/s41467-020-14337-6">10.1038/s41467-020-14337-6</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1038/s41467-020-14337-6" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		<u>Cameron CJF</u>, Dostie J and Blanchette M (2020) 
		<strong>HIFI: estimating DNA-DNA interaction frequency from Hi-C data at restriction-fragment resolution</strong>. 
		<i>Genome Biol</i>, <strong>21</strong>(1):11. 
		DOI:<a target="_blank" href="https://doi.org/10.1186/s13059-019-1913-y">10.1186/s13059-019-1913-y</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1186/s13059-019-1913-y" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Wang XQD<sup>&dagger;</sup>, <u>Cameron CJF</u><sup>&dagger;</sup>, Paquette D, Segal D, Warsaba R, Blanchette M and Dostie J (2019) 
		<strong>2C-ChIP: measuring chromatin immunoprecipitation signal from defined genomic regions with deep sequencing</strong>. 
		<i>BMC Genomics</i>, <strong>20</strong>:162. 
		DOI:<a target="_blank" href="https://doi.org/10.1186/s12864-019-5532-5">10.1186/s12864-019-5532-5</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1186/s12864-019-5532-5" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
	<li>
		Roche PJR, Gytz H, Hussain F, <u>Cameron CJF</u>, Paquette D, Blanchette M, Dostie J, Nagar B and Akavia UD (2018) 
		<strong>Double-Stranded Biotinylated Donor Enhances Homology-Directed Repair in Combination with Cas9 Monoavidin in Mammalian Cells</strong>. 
		<i>CRISPR J</i>, <strong>1</strong>(6). 
		DOI:<a target="_blank" href="https://doi.org/10.1089/crispr.2018.0045">10.1089/crispr.2018.0045</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1089/crispr.2018.0045" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span> 
	</li>
	<li>
		Malina A, <u>Cameron CJF</u>, Robert F, Blanchette M, Dostie J and Pelletier J (2015) 
		<strong>PAM multiplicity marks genomic target sites as inhibitory to CRISPR/Cas9 editing</strong>. 
		<i>Nat Commun</i>, <strong>6</strong>:10124. 
		DOI:<a target="_blank" href="https://doi.org/10.1038/ncomms10124">10.1038/ncomms10124</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1038/ncomms10124" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span> 
	</li>
	<li>
		Ma EYT, <u>Cameron CJF</u> and Kremer SC (2010) 
		<strong>Classifying and scoring of molecules with the NGN: new data sets, significance tests, and generalization</strong>. 
		<i>BMC Bioinformatics</i>, <strong>11</strong>(Suppl 8):S4. 
		DOI:<a target="_blank" href="https://doi.org/10.1186/1471-2105-11-S8-S4">10.1186/1471-2105-11-S8-S4</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1186/1471-2105-11-S8-S4" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
</ol>

<h4><strong>Peer-reviewed conference papers</strong></h4>
<ol reversed="">
	<li> 
		Kirchhof M, <u>Cameron CJF</u> and Kremer SC. 
		<strong>ABCNet: predicting chromosomal compartments directly from reference genomes</strong>. 
		<i>In preparation for IEEE BIBM 2021</i>.
	</li>
	<li>
		<u>Cameron CJF</u>, Ma EYT and Kremer SC (2010) 
		<strong>Neural Grammar Networks for Toxicology</strong>. 
		<i>2010 IEEE Symp on Computational Intelligence in Bioinformatics and Computational Biology</i> (Montréal, CAN): 1-8. 
		DOI:<a target="_blank" href="https://doi.org/10.1109/CIBCB.2010.5510322">10.1109/CIBCB.2010.5510322</a>
		<span style="float:right" data-badge-type="4" data-doi="10.1109/CIBCB.2010.5510322" data-hide-no-mentions="true" data-link-target="_blank" class="altmetric-embed"></span>
	</li>
</ol>

<h4><strong>Conferences with abstract submission</strong></h4>
<ol reversed="">
	<li><u>Cameron CJF</u>, Tagare H, Gerstein MB and Xiong Y (2020) <strong>Improving agreement between structural models and cryo-EM maps in reciprocal space</strong>. <i>5th NE Cryo-EM Symp</i>, Boston, USA</li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2018) <strong>Prediction of complete Hi-C interaction matrices from sequence-based determinants</strong>. <i>IRIC 2018 Annual Symp</i>, Montréal, CAN </li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence</strong>. <i>SCS and RegSys COSI at ISMB 2018</i>, Chicago, USA</li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2018) <strong>Prediction of complete Hi-C contact maps from genomic sequence alone</strong>. <i>RECOMB 2018</i>, Paris, FRA</li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2017) <strong>High-resolution estimation of true DNA-DNA interaction frequency from Hi-C data</strong>. <i>SCS and RegGen COSI at ISMB/ECCB 2017</i>, Prague, CZE</li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>MRCCT - 1st International Symp on Immunogenetics of Infectious and Inflammatory Diseases</i>, Montréal, CAN (PhD poster prize)</li>
	<li><u>Cameron CJF</u>, Blanchette M, De Hoon M and Dostie J (2016) <strong>Identifying ncRNA Drivers of Architectural Change in Chromatin</strong>. <i>RNA2016</i>, Kyoto, JPN</li>
	<li><u>Cameron CJF</u>, Dostie J and Blanchette M (2016) <strong>Improved Hi-C Contact Maps by Adaptive Density Estimation</strong>. <i>GLBIO/CCBC 2016</i>, Toronto, CAN</li>
	<li><u>Cameron CJF</u>, Kaplan M, Drouin A and Blanchette M (2016) <strong>Linking Transposable Elements to Chromatin Architecture in <i>Arabidopsis thaliana</i></strong>. <i>GLBIO/CCBC 2016</i>, Toronto, CAN</li>
	<li><u>Cameron CJF</u>, Fraser J, Dostie J and Blanchette M (2014) <strong>Chromosome conformation capture data improves long-range eQTL prediction</strong>. <i>2014 CSHL Conference on Epigenetics &amp; Chromatin</i>, Huntington, USA</li>
</ol>

<h4><strong>Book chapters, reviews and non-refereed publications</strong></h4>
<ol reversed="">
	<li>Wang XQD, <u>Cameron CJF</u>, Segal D, Paquette D, Blanchette M and Dostie J (2021) <strong>Profiling Chromatin Landscape at High Resolution and Throughput with 2C-ChIP. In: Bodega B., Lanzuolo C. (eds) Capturing Chromosome Conformation. Methods in Molecular Biology, vol 2157</strong>. <i>Humana, New York, NY</i>. DOI:<a target="_blank" href="https://doi.org/10.1007/978-1-0716-0664-3_8">10.1007/978-1-0716-0664-3_8</a></li>
	<li><u>Cameron CJF</u>, Fraser J, Blanchette M and Dostie J (2016) <strong>Mapping and Visualizing Spatial Genome Organization. In: Bazett-Jones D, Dellaire G (eds) The Functional Nucleus</strong>. <i>Springer, Cham</i>. DOI:<a target="_blank" href="https://doi.org/10.1007/978-3-319-38882-3_16">10.1007/978-3-319-38882-3_16</a></li>
	<li>Marhon SA, <u>Cameron CJF</u> and Kremer SC (2013) <strong>Recurrent Neural Networks. In: Bianchini M, Maggini M, Jain L (eds) Handbook on Neural Information Processing</strong>. <i>Intelligent Systems Reference Library, vol 49. Springer, Berlin, Heidelberg</i>. DOI:<a target="_blank" href="https://doi.org/10.1007/978-3-642-36657-4_2">10.1007/978-3-642-36657-4_2</a></li>
</ol>

<h4><strong>Invited talks</strong></h4>
<ol reversed="">
	<li>(July 2017) <strong>The future of Hi-C</strong>. <i>Birds of a Feather panel discussion at ISMB/ECCB 2017</i>, Prague, CZE</li>
	<li>(April 2017) <strong>High-resolution estimation of DNA-DNA contact frequency from Hi-C data</strong>. <i>The Physical Basis of Functional Genome Organization: Genome organization as viewed by molecular and visual techniques</i>, Holetown, BRB</li>
	<li>(August 2016) <strong>The 3D Genome</strong>. <i>Beijing Institute of Genomics (BIG) at the Chinese Academy of Sciences</i>, Beijing, CHN</li>
</ol>

<h4><strong>Awards and scholarships</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017&ndash;19</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Sumner Fellowship, <i>Walter Sumner Foundation</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2018</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Travel Fellowship Award for ISCB Student Council Symposium, <i>ISCB SCS</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Research Enhancement and Travel Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Graduate Mobility Award, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2016</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Mitacs Globalink Research Award, <i>Mitacs/Japan Society for the Promotion of Science</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013&ndash;14</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Grad Excellence Award in Computer Science, <i>McGill University</i></td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2013&ndash;14</strong></td>
		<td style="border-style: none; padding-bottom: 0;">Walter Hitschfeld Award - Computer Science, <i>McGill University</i></td>
	</tr>
</table>

<h4><strong>Academic service</strong></h4>
<table style="border-collapse: collapse;" cellspacing="0" cellpadding="0">
	<colgroup>
		<col width="15%" />
		<col width="85%" />
	</colgroup>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2021&ndash;22</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><a target="_blank" href="https://ypa.yale.edu/what-ypa/symposium-committee">YPA symposium</a> committee member (2021) &amp; co-coordinator (2022) </td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2020&ndash;Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><a target="_blank" href="https://ccameron.github.io/downloads/Cameron_CCTP_completed_certificate_2020.pdf">CCTP completion</a> &amp; <a target="_blank" href="https://www.cirtl.net">CIRTL</a> associate</td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2020</strong></td>
		<td style="border-style: none; padding-bottom: 0;"> <a target="_blank" href="https://www.iscb.org/ismb2020">Intelligent Systems for Molecular Biology (ISMB)</a> peer reviewer</td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>Fall 2017</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><a target="_black" href="https://ccameron.github.io/teaching/#course-lecturing">COMP 364: Computer Tools for Life Sciences</a> co-instructor (with <a target="_blank" href="http://cgoliver.com/">Oliver CG</a>)</td>
	</tr>
	<tr>
		<td style="border-style: none; padding-bottom: 0;"><strong>2016&ndash;Now</strong></td>
		<td style="border-style: none; padding-bottom: 0;"><a target="_black" href="https://ccameron.github.io/teaching/">Undergraduate and graduate student mentorship</a> (<i>N</i> = 14)</td>
	</tr>
</table>

  </article>

  

  

</div>

      </div>
    </div>

    <footer>

  <div class="wrapper" style="width: 100%;text-align: center;" >
    &copy; Copyright 2021 .
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme.

    
  </div>

</footer>


    <!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.12.4.min.js"></script>

<!-- Load Common JS -->
<script src="https://ccameron.github.io/assets/js/common.js"></script>





<!-- Include custom icon fonts -->
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/font-awesome.min.css">
<link rel="stylesheet" href="https://ccameron.github.io/assets/css/academicons.min.css">

<!-- Google Analytics
 <script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-111964338-1', 'auto');
ga('send', 'pageview');
</script> -->

<!-- Altmetric -->
<script src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>

  </body>

</html>
